backend/packages/Upgrade/src/loaders/typeormLoader.ts

```diff
@@ -1,23 +1,187 @@
+import { archivedState1692936809279 } from '../database/migrations/1692936809279-archivedState';
+import { stratificationFactorStatus1696829429134 } from '../database/migrations/1696829429134-stratificationFactorStatus';
+import { stratificationFactorFeature1696498128121 } from '../database/migrations/1696498128121-stratificationFactorFeature';
+import { addGroupIdForIndividualExclusion1710484793070 } from '../database/migrations/1710484793070-addGroupIdForIndividualExclusion';
+import { userDefaultRoleReader1713260614311 } from '../database/migrations/1713260614311-userDefaultRoleReader';
+import { Typeorm1719738784139 } from '../database/migrations/1719738784139-Typeorm';
+import { featureFlagSegmentInclusionExclusion1711566460836 } from '../database/migrations/1711566460836-featureFlagSegmentInclusionExclusion';
+import { featureFlagExposure1711569269846 } from '../database/migrations/1711569269846-featureFlagExposure';
+import { featureFlagContext1711569517358 } from '../database/migrations/1711569517358-featureFlagContext';
+import { featureFlagStatusTags1711652015345 } from '../database/migrations/1711652015345-featureFlagStatusTags';
+import { contextInMetric1712553037665 } from '../database/migrations/1712553037665-contextInMetric';
+import { featureFlagFilterMode1714680515570 } from '../database/migrations/1714680515570-featureFlagFilterMode';
+import { addingIndex1716191003726 } from '../database/migrations/1716191003726-addingIndex';
+import { monitoredIndexesForExport1718377498760 } from '../database/migrations/1718377498760-monitoredIndexesForExport';
+import { addTokenValidationFailedEnum1720515933833 } from '../database/migrations/1720515933833-addTokenValidationFailedEnum';
+import { featureFlagLists1720810654183 } from '../database/migrations/1720810654183-feature-flag-lists';
+import { removeGroupIdFromIndividualExclusion1721124249413 } from '../database/migrations/1721124249413-removeGroupIdFromIndividualExclusion';
+import { featureFlagFilterModeChangeDefault1722540825048 } from '../database/migrations/1722540825048-featureFlagFilterModeChangeDefault';
+
+const entities = [
+  ArchivedStats,
+  ConditionPayload,
+  DecisionPoint,
+  Experiment,
+  ExperimentAuditLog,
+  ExperimentCondition,
+  ExperimentError,
+  ExperimentSegmentExclusion,
+  ExperimentSegmentInclusion,
+  ExperimentUser,
+  ExplicitIndividualAssignment,
+  Factor,
+  FeatureFlag,
+  FeatureFlagExposure,
+  FeatureFlagSegmentExclusion,
+  FeatureFlagSegmentInclusion,
+  GroupEnrollment,
+  GroupExclusion,
+  GroupForSegment,
+  IndividualEnrollment,
+  IndividualExclusion,
+  IndividualForSegment,
+  Level,
+  LevelCombinationElement,
+  Log,
+  Metric,
+  MonitoredDecisionPoint,
+  MonitoredDecisionPointLog,
+  PreviewUser,
+  Query,
+  ScheduledJob,
+  Segment,
+  Setting,
+  StateTimeLog,
+  StratificationFactor,
+  User,
+  UserStratificationFactor,
+];
+
+export const migrations = [
```

Comment:
Same here. I was able to use `env.app.dirs.migrations` and did not have to define this variable.